.changeset/rare-plants-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nl-design-system-community/editor': minor
+'@nl-design-system-community/editor-react': minor
```

Comment:
De packages zijn nu 0.0.0, je kunt als eerste wel een major release doen zodat het vanaf dat moment over gaat op semver. Het is niet erg dat het nog geen volledig product is, het zou prima zijn dat een mvp pas gerealiseerd is bij v12.4.12.

Reply:
Ok, good to know!